.github/workflows/smoke-test.yml

```diff
@@ -11,13 +11,27 @@ defaults:
 jobs:
   smoke-test:
     name: Snapcraft
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v2
 
+      - name: Build
+        # https://github.com/snapcore/action-build/releases
+        uses: snapcore/action-build@v1
```

Comment:
Why is this action downgraded from previous v1.1.1 to v1? don't you want to upgrade to v1.1.2 here?

Reply:
We don't need to pin it to a micro release. https://github.com/snapcore/action-build#readme suggests using `v1` as well.

Reply:
ah, ok.

However the build still fails. I wonder if we are missing something to allow unified cgroups to work, or if nested unified cgroups prevent running docker nested.

Reply:
Yeah, I'm moving it to draft. I'll work on it this week.